Initialize landing for the Oct 02 contributor summit (#4537)
jenkins-infra/jenkins.io: content/events/contributor-summit/index.adoc

== References

* Registration - coming soon
* link:https://docs.google.com/document/d/1QLWXNG23ui-LvQXth3UREzOvLYgTMSZcv-El0H141a4/edit?usp=sharing[Coordination Google Doc]

> * link:https://groups.google.com/u/1/g/jenkinsci-dev/c/Tg3_pmHd5dE[Announcement in the developer mailing list]

Reviewer: ```suggestion
* link:https://community.jenkins.io/t/jenkins-contributor-summit-oct-02-2021-apac-emea/203/8[Announcement on community.jenkins.io Discourse site]
```

> * link:https://community.jenkins.io/t/jenkins-contributor-summit-oct-02-2021-apac-emea/203/8[Announcement on community.jenkins.io Discourse site]

Reviewer: Link to the full message, not the eighth message
```suggestion
* link:https://community.jenkins.io/t/jenkins-contributor-summit-oct-02-2021-apac-emea/203[Announcement on community.jenkins.io Discourse site]
```

== References

> * link:https://events.linuxfoundation.org/cdcon/register/[Registration]

Reviewer: Use a "coming soon" instead of a link tot he cdcon registration form
```suggestion
* Registration - coming soon
```

* link:https://docs.google.com/document/d/1QLWXNG23ui-LvQXth3UREzOvLYgTMSZcv-El0H141a4/edit?usp=sharing[Coordination Google Doc]
* link:https://groups.google.com/u/1/g/jenkinsci-dev/c/Tg3_pmHd5dE[Announcement in the developer mailing list]

> * link:https://events.linuxfoundation.org/cdcon/[cdCon and co-located events on June 22-24]

Reviewer: Link to DevOps World
```suggestion
* link:https://www.devopsworld.com/[DevOps World and co-located events on September 28-30]
```
